test/ui/QuerySummaryTest.ts

```diff
@@ -156,5 +202,121 @@ export function QuerySummaryTest() {
         expect($$(test.cmp.element).text()).not.toEqual(jasmine.stringMatching(/^Results.*of.*/));
       });
     });
+
+    describe('when the option enableNoResultsFoundMessage is set to true', () => {
+      it(`when the noResultsFoundMessage is not specified
+          it should return the default message`, () => {
+        test = Mock.optionsComponentSetup<QuerySummary, IQuerySummaryOptions>(QuerySummary, {
+          enableNoResultsFoundMessage: true
+        });
+        expect(test.cmp.options.noResultsFoundMessage).toBe(`No results for ${queryTag}`);
```

Comment:
It's kind of a bad test, it's mostly checking for getter/setter on the component. This is too basic, and not likely to catch real bugs.

I'd instead test that when there is no results it renders the correct message inside the component, not that the options are set properly.

If it's already tested somewhere else, then this test is useless and could be removed.

Reply:
Good point!